alfalfa_worker/jobs/openstudio/step_run.py

```diff
@@ -116,28 +116,27 @@ def advance_to_start_time(self):
         self.master_enable_bypass = False
         self.update_db()
 
-    def exchange_data(self):
-        """
-        Write inputs to simulation and get updated outputs
-        This does not advance the simulation on its own, instead
-        to advance the simulation increment ep.kstep and then call this function
-        See self.step
-        """
-        # Before step
-        inputs = self.read_write_arrays_and_prep_inputs()
-        self.ep.write(mlep.mlep_encode_real_data(2, 0, (self.ep.kStep - 1) * self.ep.deltaT, inputs))
-        # After step
+    def update_outputs_from_ep(self):
+        """Reads outputs from E+ step"""
         packet = self.ep.read()
         flag, _, outputs = mlep.mlep_decode_packet(packet)
         self.ep.outputs = outputs
-        return outputs
 
     def step(self):
         """
-        Simulate one simulation timestep
+        Write inputs to simulation and get updated outputs.
+        This will advance the simulation one timestep.
         """
+
+        # This doesn't really do anything. Energyplus does not check the timestep value coming from the external interface
         self.ep.kStep += 1
-        self.exchange_data()
+        # Begin Step
+        inputs = self.read_write_arrays_and_prep_inputs()
+        packet = mlep.mlep_encode_real_data(2, 0, (self.ep.kStep - 1) * self.ep.deltaT, inputs)
```

Comment:
should we document what the `2, 0` arguments are?

Reply:
We can. They're weird ExternalInterface constants, but it's worth adding a comment, probably.